README.md

```diff
@@ -1,34 +1,30 @@
-# EIPs [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+# Ethereum Improvement Proposals (EIPs) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
 Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.
 
-A browsable version of all current and draft EIPs can be found on [the official EIP site](http://eips.ethereum.org/).
+**Before you initiate a pull request**, please read the process document. Ideas should be thoroughly discussed on the [Ethereum Magicians forums](https://ethereum-magicians.org) first.
 
-# Contributing
+This repository tracks the ongoing status of EIPs. It contains:
 
- 1. Review [EIP-1](EIPS/eip-1.md).
- 2. Fork the repository by clicking "Fork" in the top right.
- 3. Add your EIP to your fork of the repository. There is a [template EIP here](eip-X.md).
- 4. Submit a Pull Request to Ethereum's [EIPs repository](https://github.com/ethereum/EIPs).
+- [Draft](https://eips.ethereum.org/all) proposals which intend to complete the EIP review process.
+- [Last Calls](https://eips.ethereum.org/all) for proposals that may become final (see also [RSS feed](https://eips.ethereum.org/last-call.xml)).
+- [Accepted](https://eips.ethereum.org/all) and [Deferred](https://eips.ethereum.org/all) proposals which are awaiting implementation or deployment by Ethereum client developers.
+- [Final](https://eips.ethereum.org/all) and [Active](https://eips.ethereum.org/all) proposals that are recorded.
```

Comment:
All of these links go to the same place.  From a UX point of view, having multiple links go to the same place is confusing because users tend to assume that each link is unique.  If we only have one link to provide, then consider putting it up in the leading sentence:
> This repository tracks the ongoing status of [EIPs](https://eips.ethereum.org/all)

Reply:
I am just raising the expectations on the index.md. I'll go ahead and and edit that.

Updated in 6abedf8 and 477a314